eyelid-match-ops/src/primitives/poly/test.rs

```diff
@@ -5,3 +5,6 @@ pub mod gen;
 
 #[cfg(test)]
 pub mod mul;
+
+#[cfg(any(test, feature = "benchmark"))]
```

Comment:
When `sample()` is moved, this can become:
```suggestion
#[cfg(test)]
```